package.json

```diff
@@ -10,7 +10,7 @@
     "database.com"
   ],
   "homepage": "http://github.com/jsforce/jsforce",
-  "version": "2.0.0-beta.27",
+  "version": "3.0.0-beta.0",
```

Comment:
We should probably release this as a new "channel". If someone is installing `jsforce` with the `beta` dist-tag it will break them when it gets overwritten with the v3 branch. How about `v3` or `next` instead of `beta`? We would also need to update the `pre-release-identifier` in the `manualRelease.yml` step (maybe elsewhere too)

Reply:
good catch! I'll change it to `next`